tools/devtool

```diff
@@ -830,28 +833,12 @@ cmd_test_debug() {
 # Example: `devtool fmt`
 #
 cmd_fmt() {
-
-    # Parse any command line args.
-    while [ $# -gt 0 ]; do
-        case "$1" in
-            "-h"|"--help")      { cmd_help; exit 1; } ;;
-            *)
-                die "Unknown argument: $1. Please use --help for help."
-            ;;
-        esac
-        shift
-    done
-
-    ensure_devctr
-
-    say "Applying rustfmt ..."
-    run_devctr \
-        --user "$(id -u):$(id -g)" \
-        --workdir "$CTR_FC_ROOT_DIR" \
-        -- \
-        cargo fmt --all
+    cmd_sh "cargo fmt --all"
```

Comment:
Mh, our CI uses different formatting, because it passes the contents of `tests/fmt.toml` to the `--config` flag. Maybe we should do the same here?

Reply:
True. Should we just remove this instead?

Reply:
I think if we update this to apply the formatting the same way our CI expects it to be, it can be useful to have. I could imagine updating my pre-commit hook to just call this devtool command instead of whatever I am doing in there currently.

Reply:
done